target/linux/qualcommax/patches-6.1/0141-arm64-dts-qcom-ipq8074-Add-QUP-UART6-node.patch

```diff
@@ -0,0 +1,48 @@
+From e1faf160730772cd6de302d284155ed0dce015de Mon Sep 17 00:00:00 2001
+From: Paweł Owoc <frut3k7@gmail.com>
+Date: Thu, 29 Feb 2024 21:39:05 +0100
+Subject: [PATCH] arm64: dts: qcom: ipq8074: Add QUP UART6 node
+
+Add node to support the QUP UART6 controller inside of IPQ8074.
+Used by some routers to communicate with a Bluetooth controller.
+
+Signed-off-by: Paweł Owoc <frut3k7@gmail.com>
+---
+ arch/arm64/boot/dts/qcom/ipq8074.dtsi | 19 +++++++++++++++++++
+ 1 file changed, 19 insertions(+)
+
+--- a/arch/arm64/boot/dts/qcom/ipq8074.dtsi
++++ b/arch/arm64/boot/dts/qcom/ipq8074.dtsi
+@@ -561,6 +561,13 @@
+ 				bias-disable;
+ 			};
+ 
++			serial_5_pins: serial5-state {
```

Comment:
This pinctrl should be placed in the dts of the device instead of soc.dtsi.

Reply:
Are these gpios used for other purposes for ipq8074?

Reply:
These gpios can also be used for spi.

Reply:
I found documentation for IPQ8078. `gpio23` and `gpio24` used by default for the serial port (serial_4_pins) can also be used by SPI.

Reply:
Patch is accepted: https://lore.kernel.org/all/20240229205426.232205-1-frut3k7@gmail.com/
Should I really remove the default pin configuration?